docs/docs/pricing/README.md

```diff
@@ -84,63 +99,42 @@ To discuss an Enterprise plan, please contact the Pipedream Sales Team [using th
 
 Pipedream uses a number of terms to describe platform metrics and details of our plans. See the definitions of key terms below.
 
-### Invocations
+### Credit
 
-Pipedream counts an **invocation** each time a workflow or event source is triggered by an incoming event.
+Pipedream counts your usage in terms of _credits_ of total compute time across all workflows in the billing period.
 
-Pipedream increments the count of invocations by one for each incoming event, regardless of the number of steps in your workflow. For example, if you send an HTTP request to a workflow with five steps, Pipedream will count that as one invocation.
+One credit is {{ $site.themeConfig.base_credits_price.seconds }} seconds of compute time at {{ $site.themeConfig.base_credits_price.memory }} megabytes of memory.
 
-Moreover, if you have a workflow triggered by a cron job running once a minute for the entire day, that will incur 1,440 invocations (60 minute \* 24 hours). For cron-triggered workflows or event sources, remember that you can always reduce the frequency to reduce your invocations.
+Regardless of the number of steps in your workflow, if your workflow run runs for {{ $site.themeConfig.base_credits_price.seconds }} or less seconds then that run only costs 1 credit.
 
-If an event emitted by an event source triggers a single workflow, that will count as **two** invocations: one for the source, and one for the workflow. In other words, source and workflow execution is distinct: each counts invocations on its own.
-
-Your workflow's [memory settings](/workflows/settings/#memory) also impact the number of invocations you're charged for each workflow execution. [Read more here](#how-does-workflow-memory-affect-billable-invocations).
+Adding additional memory capacity to workflows will increase credit usage in intervals of 256 megabytes. For example, doubling the memory of a workflow from {{ $site.themeConfig.base_credits_price.memory }} to {{ $site.themeConfig.base_credits_price.memory * 2 }} will double the cost of credits in the same execution time.
 
 #### Scenarios
 
-::: details Webhook triggered workflow
+::: details A standard workflow that runs 5 times with one second per run
 
-*1* invocation is incurred per HTTP webhook. The HTTP endpoint is *not* considered a source.
+Only 1 credit is incurred, because the workflow ran for a total of 5 seconds (5 runs at 1 second each = 5 total seconds of compute time)
 
 :::
 
-::: details Scheduled workflow
-
-*1* invocation is incurred per new event emitted by the source.
-*1* invocation is incurred per new event processed by the workflow.
+::: details Developing a workflow with test events in the Pipedream workflow builder
```

Comment:
I like this example to clarify that testing is free. I would also communicate that in the summary at the top of this section.